### ISSUE TITLE
Colour of railway=station areas

### ISSUE THREAD

**Issue by jc86035** [gravitystorm/openstreetmap-carto#2618]
Currently `building=train_station` is rendered brown but `railway=station` + `area=yes` is rendered pink. It would be nice if both of them could be rendered brown or pink or whatever.

(The other issue with `building=train_station` is that the brown covers any other buildings which happen to be above the station, but this is already part of #2504.)

**Comment by nebulon42** [comment]
If I understand these two tags correctly then `building=train_station` is for the station building, while `railway=station` is either a node or the area spanning the whole station. This is how the [Wiki](https://wiki.openstreetmap.org/wiki/Tag:railway%3Dstation) describes it. Under this circumstances the rendering is correct. The building gets the building brown colour and the station area gets the railway landuse colour.

In other words the building itself should not be tagged with `railway=station`.
